pkg/integrations/kafka_exporter/kafka_exporter.go

```diff
@@ -0,0 +1,170 @@
+package kafka_exporter //nolint:golint
+
+import (
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	kafka_exporter "github.com/davidmparrott/kafka_exporter/v2/exporter"
+	"github.com/go-kit/kit/log"
+	"github.com/grafana/agent/pkg/integrations"
+	"github.com/grafana/agent/pkg/integrations/config"
+)
+
+// DefaultConfig holds the default settings for the kafka_lag_exporter
+// integration.
+var DefaultConfig = Config{
+	UseSASLHandshake:        true,
+	KafkaVersion:            sarama.V2_0_0_0.String(),
+	MetadataRefreshInterval: "1m",
+	AllowConcurrent:         true,
+	MaxOffsets:              1000,
+	PruneIntervalSeconds:    30,
+	TopicsFilter:            ".*",
+	GroupFilter:             ".*",
+}
+
+// Config controls kafka_exporter
+type Config struct {
+	Common config.Common `yaml:",inline"`
+
+	// Address array (host:port) of Kafka server
+	KafkaUri []string `yaml:"kafka_uris,omitempty"`
+
+	// Connect using SASL/PLAIN
+	UseSASL bool `yaml:"use_sasl,omitempty"`
+
+	// Only set this to false if using a non-Kafka SASL proxy
+	UseSASLHandshake bool `yaml:"use_sasl_handshake,omitempty"`
+
+	// SASL user name
+	SASLUsername string `yaml:"sasl_username,omitempty"`
+
+	// SASL user password
+	SASLPassword string `yaml:"sasl_password,omitempty"`
+
+	// The SASL SCRAM SHA algorithm sha256 or sha512 as mechanism
+	SASLMechanism string `yaml:"sasl_mechanism,omitempty"`
+
+	// Connect using TLS
+	UseTLS bool `yaml:"use_tls,omitempty"`
+
+	// The optional certificate authority file for TLS client authentication
+	TLSCAFile string `yaml:"tls_cafile,omitempty"`
+
+	// The optional certificate file for TLS client authentication
+	TLSCertFile string `yam:"tls_certfile,omitempty"`
```

Comment:
(Also notice the `yam` typo here)

```suggestion
	CertFile string `yaml:"cert_file,omitempty"`
```